src/server/usage/classes/collector_set.js

```diff
@@ -117,19 +117,23 @@ export class CollectorSet {
       }
     };
 
-    return Object.keys(apiData).reduce((accum, currName) => {
-      const value = apiData[currName];
-
-      let newName = currName;
-      newName = snakeCase(newName);
-      newName = newName.replace(/^(1|5|15)_m/, '$1m'); // os.load.15m, os.load.5m, os.load.1m
-      newName = newName.replace('_in_bytes', '_bytes');
-      newName = newName.replace('_in_millis', '_ms');
-
-      return {
-        ...accum,
-        [newName]: getValueOrRecurse(value),
-      };
-    }, {});
+    if (!Array.isArray(apiData)) {
+      return Object.keys(apiData).reduce((accum, field) => {
+        const value = apiData[field];
+        let newName = field;
+
+        newName = snakeCase(newName);
+        newName = newName.replace(/^(1|5|15)_m/, '$1m'); // os.load.15m, os.load.5m, os.load.1m
+        newName = newName.replace('_in_bytes', '_bytes');
+        newName = newName.replace('_in_millis', '_ms');
+
+        return {
+          ...accum,
+          [newName]: getValueOrRecurse(value),
+        };
+      }, {});
+    } else {
+      return apiData;
```

Comment:
Per the comments from @ycombinator about the test, it seems weird that we aren't doing any manipulation if it's an array since it seems reasonable it could be an array of objects that need to be manipulated?

Reply:
f2474ee